core/src/com/po/fuck/model/Core.java

```diff
@@ -13,28 +13,29 @@
 import com.po.fuck.model.position.GeometryData;
 
 public class Core implements Updatable {
-    static { // TODO: remove
-        forceInit(Destructable.class);
-        forceInit(All.class);
-    }
-    
-    public static Managed<Player> player;
+    public Managed<Player> player;
     public static Managed<Coins> coinsCounter;
     public static Managed<ObjectFollower> objectFollower;
```

Comment:
Probably make non-static as well?

Reply:
Not very obvious task, because `coinsCounter` accessed from the `Entity`, and `objectFollower` needed for the controller to calculate global position.

Reply:
Probably, problably, we could move the `objectFollower` into `Controller`. Then pass the object to follow and return calculated position in accordance with `objectFollower`?

Reply:
`objectFollower` is not a `Controller`'s responsibility.